matrixbot/matrix.py

```diff
@@ -713,28 +729,44 @@ def get_rooms(self):
     def get_room_aliases(self, room_id):
         return self.room_aliases[room_id] if room_id in self.room_aliases else []
 
-    def sync(self, ignore=False, timeout_ms=30000):
-        response = self.client.api.sync(self.sync_token, timeout_ms, full_state='true')
-        self._set_rooms(response)
-        self.sync_token = response["next_batch"]
-        self.logger.info("!!! sync_token: %s" % (self.sync_token))
-        self.logger.log(EXTRA_DEBUG, "Sync response: %s" % (response))
+    async def _dispatch(self, response):
+        _tasks = []
 
+        if not response:
+            return
+
+        async def _(plugin, callback):
+            try:
+                plugin.dispatch(callback)
+            except Exception as e:
+                self.logger.error(
+                    "Error in plugin %s: %s" % (plugin.name, e)
+                )
+
+        for plugin in self.plugins:
+            _tasks.append(asyncio.create_task(_(plugin, self.send_message)))
+        _tasks.append(asyncio.ensure_future(
```

Comment:
fix alignment for _tasks.append's ? maybe some style and quality tool could complain here

Reply:
I don't see any alignment issue. Note that L747 is into the `for` loop but not the L748

Reply:
I think it was about one tab/spaces issue but it is ok on my side so forget about it. Thanks for commenting.